BSP/STM32F413/Src/BSP_CAN.c

```diff
@@ -2,10 +2,13 @@
 #include "stm32f4xx.h"
 
 #define CAN_MODE        CAN_Mode_Normal
+#define CANFIFOMAX  32 //most messages it can take before wrapping around
 
 static CanTxMsg TxMessage;
 static CanRxMsg RxMessage;
-
+uint8_t CANFIFO[32][8] = {0}; //initialize 32 by 8 array containing FIFO data 
```

Comment:
Since this isn't `const`, name it something like `rxFifo` instead, and make it static.